e2e/cypress/integration/channel/channel_switcher_spec.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel @smoke
+
+describe('Channel Switcher', () => {
+    let testTeam;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+
+            // # Add some channels
+            cy.apiCreateChannel(testTeam.id, 'channel-a', 'SwitchChannel A', 'O');
+            cy.apiCreateChannel(testTeam.id, 'channel-b', 'SwitchChannel B', 'O');
+            cy.apiCreateChannel(testTeam.id, 'channel-c', 'SwitchChannel C', 'O');
+
+            // # Login as test user and go to town square
+            cy.apiLogin(user);
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2031 - should switch channels by keyboard', () => {
```

Comment:
Hi @reflog , we usually have 1-to-1 mapping of our TM4J test steps to cypress test cases for accurate reporting. Please assign the test cases associated with the test TM4J test steps, such that `MM-T2031_<step number>`. 

If you deem 1-to-1 mapping is not achievable, please comment on the ones you combined so we can update the TM4J test case to reflect the same.

Thanks.

```suggestion
    it('MM-T2031_1 - should switch channels by keyboard', () => {
```